cmd/process-agent/api/secrets.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package api
+
+import (
+	"net/http"
+)
+
+func secretRefreshHandler(deps APIServerDeps, w http.ResponseWriter, _ *http.Request) {
+	response, err := deps.Secrets.Refresh()
+	if err != nil {
+		deps.Log.Errorf("error while refresing secrets: %s", err)
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
It doesn't look like the error is JSON, why are we setting the content-type?

Reply:
in `writeError`, we write the error as a json. So it seemed like the correct content-type. Am I mistaken?
https://github.com/DataDog/datadog-agent/blob/3f380282ecd07c052d1133eac27bab372bdba5a9/cmd/process-agent/api/util.go#L14

Reply:
Good idea. Done!